SQF/dayz_code/system/zombie_agent.fsm

```diff
@@ -700,7 +701,7 @@ class FSM
   initState="init";
   finalStates[] =
   {
-    "End"
+    "End",
```

Comment:
Not sure why, but this change was made in vanilla DayZ 1.8.5 too:
https://github.com/DayZMod/DayZ/blob/Development/SQF/dayz_code/system/zombie_agent.fsm

So it definitely does not error.